### ISSUE TITLE
Sugerencias varias

### ISSUE THREAD

**Issue by mcjarod** [diniremix/ControlParking#2]
Buenas tengo algunas sugerencias

- Que se ponga automáticamente en mayúsculas la placas
- No debería poder ingresar más de una vez el mismo vehículo si no ha salido.
- Al buscar placa de un vehículo en la sección de ingresos no aparecen resultados.
- En el reporte diario no aparecen los vehículos que solo se han ingresado que serian los que están en el parqueadero.
- La fecha y hora debe ser del momento en que presiono el botón "ingreso" y no del momento en que se cargó la página.
- No aparece donde configurar iva, los cobros según el tiempo o según algún convenio, o facturas de mensualidades, vetar vehículos, cantidad de parqueaderos.
- No aparece donde imprimir recibo de ingreso, factura de salida, cierre de caja.

Estaré presto a hacer las pruebas, un saludo y felicitaciones por comenzarlo.

**Comment by diniremix** [comment]
Gracias por el interes en la app.
Antes de revisar a menudo los cambios y sugerencias, propongo algunas cosas antes:
- Revisar de forma completa el estado actual de la aplicación y rescatar lo esencial para dar paso a una versión más moderna, y con nuevas funcionalidades.
- Separar html y js (Frontend), de php (Backend).
- Utilizar un framework para construir el Frontend (Bootstrap, Materializecss, Angular, etc).
- De igual manera con PHP, utilizar herramientas para agilizar su desarrollo (Slim PHP, Laravel/lumen etc).

Cualquier sugerencia para atender éstas, es siempre bienvenida.

**Comment by FabioMontenegro** [comment]
Hola, me parece un software muy interesante. Pero no encuentro la opción de configuración de tarifas. Será que me falto realizar algún paso?

**Comment by diniremix** [comment]
@stultitia1 
No, la opción de configuración de tarifas, solo esta disponible a nivel de **Base de Datos**
